alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOCustomConfig.java

```diff
@@ -88,6 +91,24 @@
     @Value("${transform.core.version}")
     private String coreVersion;
 
+    @Value("${transform.core.tika.exifTool.windowsOS}")
+    private String exifToolCommandOnWindows;
+
+    @Value("${transform.core.tika.exifTool.unixOS}")
+    private String exifToolCommandOnUnix;
+
+    @Bean("exifTool")

```

Comment:
I don't see this bean is used anywhere.

Reply:
"exifTool" removed. Thanks for pointing that out!